Pterodactyl/ScatterDataGH.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Grasshopper.Kernel;
+using PterodactylCharts;
+
+namespace Pterodactyl
+{
+    public class ScatterDataGH : GH_Component
+    {
+        public ScatterDataGH()
+  : base("Scatter Data", "Scatter Data",
+      "Add point data",
+      "Pterodactyl", "Advanced Graph")
+        {
+        }
+        protected override void RegisterInputParams(GH_Component.GH_InputParamManager pManager)
+        {
+            pManager.AddColourParameter("Palette", "Palette", "Add a list of colors to map the Values. Maybe any number from 1 to 4096", GH_ParamAccess.list, new List<Color> {Color.Black, Color.Orange, Color.Red});
+            pManager.AddIntegerParameter("Marker", "Marker", "Choose point marker: 0 - None, 1 - Circle, 2 - Square, 3 - Diamond, 4 - Triangle, 5 - Cross, 6 - Plus", GH_ParamAccess.item, 1);
```

Comment:
I think you need to create Markers (list) here, because when I applied 2 or more values, then I got an exception that it should fit the count of the Markers

Reply:
Or maybe Values means something else that I thought

Reply:
It is marker because I wanted to make it only one style for all related x and y values. Could be different but that can be created with another scatter data? I think that maybe you got an exception  of scatter values and marker sizes should be the same. Yeah I will provide more descriptive text.